LayoutKitTests/ReloadableViewLayoutAdapterTestCase.swift

```diff
@@ -446,6 +451,8 @@ class ReloadableViewLayoutAdapterTestCase: XCTestCase {
         var batchUpdates = BatchUpdates()
         batchUpdates.insertItems.append(IndexPath(item: 2, section: 1))
         batchUpdates.insertSections.insert(2)
+        batchUpdates.reloadItems.append(IndexPath(item: 1, section: 0))
+        batchUpdates.reloadSections.insert(1)
```

Comment:
this test does both an insert and a reload on section 1, so I am worried that the reload behavior could mask a bug in the insert behavior.

To completely isolate the tests I would prefer it if you added a new section that is only used for testing section reloading.